src/svgcontext.ts

```diff
@@ -113,18 +167,21 @@ export class SVGContext implements RenderContext {
     };
 
     this.state_stack = [];
-
-    // Test for Internet Explorer
-    this.iePolyfill();
   }
 
+  create(svgElementType: 'g'): SVGGElement;
```

Comment:
Can we add a comment along the lines of:
```
  /**
   * Use one of the overload signatures to create an SVG element of a specific type.
   * The last overload accepts an arbitrary string, and is identical to the implementation signature.
   */
```
It's probably redundant for TS gurus, but this syntax was new to me and a comment would have helped me for sure.

Reply:
Done